### PR TITLE
Tweak code owners and dependabot config

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

**Review by b1ron — APPROVED** [review]
Looks good!

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -6,9 +6,10 @@
 /cmd/                @FerretDB/leads @FerretDB/specops
 /internal/           @FerretDB/leads @FerretDB/specops
 
+/tests/              @b1ron
```
Comment (rumyantseva) [inline .github/CODEOWNERS:9]: Is it enough to only have @b1ron?

**Review by rumyantseva — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (AlekSi) [reply .github/CODEOWNERS:9]: It is less confusing this way for us

Comment (rumyantseva) [reply .github/CODEOWNERS:9]: 👌

**Pull request merged**